<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleVideoLicovani</class>
 <widget class="QWidget" name="SingleVideoLicovani">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="vybraneVideo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="prubehVypoctu">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>250</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ulozitPB">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Uložit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="slozkaParametruLE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="slicovatPB">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Slícovat</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>131</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="vyberVidea">
      <property name="text">
       <string>Výběr videa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="slozkaParametruPB">
      <property name="text">
       <string>Složka s parametry</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>103</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="zobrazVysledekLicovaniPB">
      <property name="text">
       <string>Zobrazit slícované
 video</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="grafickeHodnoceniPB">
      <property name="text">
       <string>Grafické hodnocení</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
